lib/ProMotion/table/data/table_data.rb

```diff
@@ -59,8 +59,13 @@ def filtered?
       @filtered == true
     end
 
+    def search_string
+      @search_string ||= null
+    end
+
     def search(search_string)
       @filtered = true
+      @search_string = search_string
```

Comment:
Does this still provide the correct value? For some reason I thought this gets overwritten (or is not up-to-date) somewhere later on.

Reply:
There were some significant changes that had to happen in order to support searching on iOS 8+ so either some of that functionality got lost on accident or couldn't be supported.